toolkit/tools/pkg/specarchchecker/specarchchecker.go

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// A tool for filtering specs by architecture.
+
+package specarchchecker
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/logger"
+	"github.com/microsoft/CBL-Mariner/toolkit/tools/internal/rpm"
+	simpletoolchroot "github.com/microsoft/CBL-Mariner/toolkit/tools/pkg/simpletoolchroot"
+)
+
+type ArchChecker struct {
+	simpleToolChroot simpletoolchroot.SimpleToolChroot
+}
+
+const (
+	chrootName = "specarchchecker_chroot"
+	runChecks  = false
+)
```

Comment:
For consts, that are used only in one function, could we move them into the scope of the function and out of the global scope?